src/averagePerceptron.jl

```diff
@@ -0,0 +1,102 @@
+"""
+This module contains the Average Perceptron model
+which was original implemented by Matthew Honnibal.
+
+The model learns by basic perceptron algorithm
+but after all iterations weights are being averaged
+"""
+module AveragePerceptron
+    using DelimitedFiles
+    using DataStructures
+
+    mutable struct initModel
+            classes
+            weights
+            _totals
+            _tstamps
+            i
+            START
+        end
+
+    self = initModel(OrderedSet(), Dict(), DefaultDict(0), DefaultDict(0), 1, ["-START-", "-START2-"])
+
+    function predict(features)
+        """Predicting the class using current weights by doing Dot-product of features
+        and weights and return the scores"""
+        global self

```

Comment:
What's this for? Have you written this based on some python code?

Reply:
Yes, I tried to implement it like as in python I would. But the is wrong and I am now using struct instead of module and I am merging both files.
I have one thought in functions in sentiment.jl like:
`function(m::SentimentModel)(text::Array{T, 1}) where T <: AbstractString
    return get_sentiment(text, m.weight, m.words)
end`
what is the use of these type of functions ??

Reply:
This kind of a function definition allows us to use a model object a function to do the inference. So we can write code like
```
m=SentimentModel() #loads saved weights
m(input_data)      #performs inference
```

Reply:
Oh, That means that function associated to all the objects ?